src/components/Checkout/PaymentWrapper.vue

```diff
@@ -61,9 +65,12 @@ export default {
 					key: 'pp',
 				},
 			],
-			selectedOption: 'pp',
+			selectedOption: 'bt',
```

Comment:
We now default to Braintree unless we know they used PayPal in their last transaction.